src/machine/board_nano-rp2040.go

```diff
@@ -0,0 +1,99 @@
+// +build nano_rp2040
+
+// This contains the pin mappings for the Arduino Nano RP2040 Connect board.
+//
+// Sometimes the board is not detected even when the board is connected to your computer.
+// To solve this, place a jumper wire between the REC and GND pins, then connect the board to your computer.
+//
+// For more information, see: https://store.arduino.cc/nano-rp2040-connect
+// Also
+// - Datasheets: https://docs.arduino.cc/hardware/nano-rp2040-connect
+// - Nano RP2040 Connect technical reference: https://docs.arduino.cc/tutorials/nano-rp2040-connect/rp2040-01-technical-reference
+//
+package machine
+
+// Digital Pins
+const (
+	D2  Pin = GPIO25
+	D3  Pin = GPIO15
+	D4  Pin = GPIO16
+	D5  Pin = GPIO17
+	D6  Pin = GPIO18
+	D7  Pin = GPIO19
+	D8  Pin = GPIO20
+	D9  Pin = GPIO21
+	D10 Pin = GPIO5
+	D11 Pin = GPIO7
+	D12 Pin = GPIO4
+	D13 Pin = GPIO6
+	D14 Pin = GPIO26
+	D15 Pin = GPIO27
+	D16 Pin = GPIO28
+	D17 Pin = GPIO29
+	D18 Pin = GPIO12
+	D19 Pin = GPIO13
+)
+
+// Analog pins
+const (
+	A0 Pin = GPIO26
```

Comment:
Do you perhaps want to use

```go
A0 = ADC0 
```

Since they are also already defined in `machine_rp2040.go`?

Reply:
I was just looking at pin diagram for Arduino Nano RP2040 Connect board itself.
Does it make any difference?

<img width="345" alt="Nano-RP2040_analog-pins" src="https://user-images.githubusercontent.com/1061565/121353278-f79e7380-c92d-11eb-926f-e9e07389c43f.png">

Reply:
I mean will ADC code fail in this case? I see it checks in `switch` for `ADCX`, will it fail checking against `GPIO26` even if they have same int values?

Reply:
nvm, changed that. I was just curios would it make any difference.